code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -140,10 +140,13 @@
 			LateChoices()
 			return
 
-		if(SSticker.queued_players.len || (relevant_cap && living_player_count() >= relevant_cap && !(ckey(key) in GLOB.admin_datums)))
+		if(SSticker.queued_players.len || (relevant_cap && living_player_count() >= relevant_cap))
 			if(IS_PATRON(src.ckey))
 				LateChoices()
 				return
+			if(client in GLOB.admins)
+				LateChoices()
+				return
```

Comment:
```suggestion
			if(IS_PATRON(src.ckey) || client in GLOB.admins)
				LateChoices()
				return
```

~~Fastest review in Bee~~

Reply:
ew I dont really like combining but I guess

Reply:
```suggestion
			if(IS_PATRON(src.ckey) || (client in GLOB.admins))
				LateChoices()
				return
```